grakn-engine/src/main/java/ai/grakn/engine/GraknCreator.java

```diff
@@ -172,6 +180,22 @@ protected static TaskManager taskManager(GraknEngineConfig config, EngineGraknTx
         return new RedisTaskManager(engineId, config, jedisPool, consumers, factory, lockProvider, metricRegistry);
     }
 
+    @VisibleForTesting
```

Comment:
what is this witchcraft

Reply:
Nothing special. Just makes it obvious that this is exclusively exposed for testing only.